packages/components/src/Checkbox/Checkbox/Checkbox.module.scss

```diff
@@ -10,11 +10,11 @@ $dt-color-checkbox-background-color-checked: $color-gray-500;
 
 // override bootstrap styles
 .checkbox.checkbox {
-  @include form-input-visually-hidden;
-
   width: $checkbox-size;
   height: $checkbox-size;
   margin: 0;
+
+  @include form-input-visually-hidden;
```

Comment:
Wow, we have some super old style rules in there if we're overriding bootstrap 😆 
<img width="420" alt="Screenshot 2024-10-17 at 2 00 51 PM" src="https://github.com/user-attachments/assets/304f2d29-dc4a-485d-aa90-4832bf5b194a">

Reply:
heheh, that is to say I don't think we need to 🪓  that in this PR

Reply:
Yeah we have some pretty odd things 😬 But best to remove it separately if we want to :)